{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "/home/cike/code/src",
    "dependencies": [
      "munch==2.5.0",
      "numpy==1.21.6",
      "PyYAML==5.4.1",
      "sacred==0.8.2",
      "torch==1.10.2"
    ],
    "mainfile": "meta_main.py",
    "name": "pymarl",
    "repositories": [],
    "sources": [
      [
        "meta_main.py",
        "_sources/meta_main_25498bb0d3299910d269250f30221af0.py"
      ],
      [
        "utils/config_utils.py",
        "_sources/config_utils_11a418cc99d65a52a4ab3d9c9e93f913.py"
      ],
      [
        "utils/logging.py",
        "_sources/logging_867f86b10403f8b7e6826558d04fb18e.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/cike/anaconda3/envs/pymarl/lib/python3.7/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"src/meta_main.py\", line 43, in my_main\n    run(_run, config, _log)\n",
    "  File \"src/meta_main.py\", line 79, in run\n    pop.run()\n",
    "  File \"/home/cike/code/src/meta/population/stage1_population.py\", line 159, in run\n    self.train_inner_all()\n",
    "  File \"/home/cike/code/src/meta/population/stage1_population.py\", line 83, in train_inner_all\n    self.train_inner(ind_id)\n",
    "  File \"/home/cike/code/src/meta/population/stage1_population.py\", line 77, in train_inner\n    self.dones[ind_id] = self.individuals[ind_id].train()\n",
    "  File \"/home/cike/code/src/meta/individual/stage1_individual.py\", line 110, in train\n    self.learner.train(episode_sample, self.runner.t_env, self.episode)\n",
    "  File \"/home/cike/code/src/learners/q_learner.py\", line 56, in train\n    target_agent_outs = self.target_mac.forward(batch, t=t)\n",
    "  File \"/home/cike/code/src/controllers/basic_controller.py\", line 34, in forward\n    **kwargs)\n",
    "  File \"/home/cike/code/src/modules/agents/rnn_agent.py\", line 24, in forward\n    h = self.rnn(x, h_in)\n",
    "  File \"/home/cike/anaconda3/envs/pymarl/lib/python3.7/site-packages/torch/nn/modules/module.py\", line 1102, in _call_impl\n    return forward_call(*input, **kwargs)\n",
    "  File \"/home/cike/anaconda3/envs/pymarl/lib/python3.7/site-packages/torch/nn/modules/rnn.py\", line 1147, in forward\n    self.bias_ih, self.bias_hh,\n",
    "RuntimeError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 47.46 GiB total capacity; 4.82 GiB already allocated; 6.31 MiB free; 5.04 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF\n"
  ],
  "heartbeat": "2024-10-21T07:33:21.844969",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz",
    "gpus": {
      "driver_version": "470.182.03",
      "gpus": [
        {
          "model": "Quadro RTX 8000",
          "persistence_mode": false,
          "total_memory": 48601
        },
        {
          "model": "Tesla P100-PCIE-16GB",
          "persistence_mode": false,
          "total_memory": 16280
        },
        {
          "model": "Tesla P100-PCIE-16GB",
          "persistence_mode": false,
          "total_memory": 16280
        },
        {
          "model": "Tesla P100-PCIE-16GB",
          "persistence_mode": false,
          "total_memory": 16280
        },
        {
          "model": "Tesla P100-PCIE-16GB",
          "persistence_mode": false,
          "total_memory": 16280
        }
      ]
    },
    "hostname": "89ddac7113b8",
    "os": [
      "Linux",
      "Linux-4.15.0-213-generic-x86_64-with-debian-buster-sid"
    ],
    "python_version": "3.7.12"
  },
  "meta": {
    "command": "my_main",
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2024-10-21T07:14:35.893142",
  "status": "FAILED",
  "stop_time": "2024-10-21T07:33:21.851353"
}